### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by rushijagani**
Mobile Menu styling improved for all header layouts

### CODE REVIEW

File: inc/template-parts.php [hunk 1]
```diff
@@ -151,24 +151,26 @@ function ast_masthead_custom_nav_menu_items( $items, $args ) {
 	 * @since 1.0.0
 	 */
 	function ast_masthead_toggle_buttons_primary() {
+
+		$menu_title = apply_filters( 'ast_main_menu_toggle_label', __( 'Menu', 'astra' ) );
```
Comment (maheshwaghmare) [inline inc/template-parts.php]: @rushijagani Remove filter `ast_main_menu_toggle_label` which is defined below.

File: sass/site/navigation/_primary-menu.scss [hunk 2]
```diff
@@ -145,12 +145,26 @@
 	}
 
 	.main-header-menu {
+		background-color: #ececec;
```
Comment (maheshwaghmare) [inline sass/site/navigation/_primary-menu.scss]: @rushijagani Use SASS color variable instead of hard coded color code `#ececec`.

Comment (rushijagani) [inline sass/site/navigation/_primary-menu.scss]: Used color variable for submenu border & mobile header menu background color.

File: inc/template-parts.php (hunk 1 above)
Comment (rushijagani) [inline inc/template-parts.php]: Removed unwanted filter.

**Pull request merged**